launch/hsi_color_filter.launch

```diff
@@ -0,0 +1,36 @@
+<launch>
+  <arg name="node_name" default="hsi_color_filter" />
+
+  <arg name="image" default="image" doc="The image topic. Should be remapped to the name of the real image topic." />
```

Comment:
default should be `"/image_publisher/output"` to subscribe image_publisher topic in default.